{{cookiecutter.project_name}}/tests/test_switch.py

```diff
@@ -0,0 +1,43 @@
+"""Test {{cookiecutter.friendly_name}} switch."""
+from homeassistant.components.switch import SERVICE_TURN_OFF, SERVICE_TURN_ON
+from homeassistant.const import ATTR_ENTITY_ID
+from pytest_homeassistant_custom_component.async_mock import call, patch
+from pytest_homeassistant_custom_component.common import MockConfigEntry
```

Comment:
`async_mock` module  no more in `pytest_homeassistant_custom_component` package.

```suggestion
from unittest.mock import call
from unittest.mock import patch

from homeassistant.components.switch import SERVICE_TURN_OFF, SERVICE_TURN_ON
from homeassistant.const import ATTR_ENTITY_ID
from pytest_homeassistant_custom_component.common import MockConfigEntry
```